code/controllers/subsystem/security_level.dm

```diff
@@ -56,6 +56,13 @@ SUBSYSTEM_DEF(security_level)
 		SSshuttle.emergency.modTimer(old_shuttle_call_time_mod)
 		SSshuttle.emergency.modTimer(selected_level.shuttle_call_time_mod)
 
+	//SKYRAT EDIT ADDITION START- In the event of an orange alert, change the airlock permissions
+	if (istype(current_security_level, /datum/security_level/orange))
+		enable_engineering_access()
```

Comment:
Orange alert should really inform engineers that this is a thing, otherwise they'll look at the new flashing doors and be confused

Reply:
Added an announcement.